pycbc/detector.py

```diff
@@ -445,3 +445,29 @@ def time_delay_from_earth_center(self, right_ascension, declination, t_gps):
             np.array([np.float32(earth.x), np.float32(earth.y),
                       np.float32(earth.z)]), right_ascension,
             declination, t_gps)
+
+    def arm_response(self, ref_time, wave):
+        """ Returns response
+        Parameters
+        ----------
+        ref_time : numpy.ScalarType
+                  Reference time
+        wave : numpy.ndarray shape (2,3)
+        Returns
+        -------
+        n_r = numpy.ndarray shape (3,3)
+          position unit vectors with each row corrsp to single det"""
+        
+        pos = self.get_pos(t_ref)
```

Comment:
To make things easy to match up and ensure correctness, it might be good to use the same notation as for the standard equations.